### PR TITLE
Move modules to independent repositories.

### PR DESCRIPTION
**Pull request by ActoryOu**
Remove modules folder in this repo, and update related documants.
- BG96: https://github.com/FreeRTOS/FreeRTOS-Cellular-Interface-Reference-Quectel-BG96
- HL7802: https://github.com/FreeRTOS/FreeRTOS-Cellular-Interface-Reference-Sierra-Wireless-HL7802
- SARA-R4: https://github.com/FreeRTOS/FreeRTOS-Cellular-Interface-Reference-ublox-SARA-R4
- Community: https://github.com/FreeRTOS/FreeRTOS-Cellular-Interface-Community-Supported-Ports

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,9 +75,9 @@ It is recommended that you start by cloning the implementation of one of the exi
 
  Current Example Implementations:
 
-* [Quectel BG96](https://github.com/FreeRTOS/FreeRTOS-Cellular-Interface/tree/main/modules/bg96)
-* [Sierra Wireless HL7802](https://github.com/FreeRTOS/FreeRTOS-Cellular-Interface/tree/main/modules/hl7802)
-* [U-Blox Sara-R4](https://github.com/FreeRTOS/FreeRTOS-Cellular-Interface/tree/main/modules/sara_r4)
+* [Quectel BG96](https://github.com/FreeRTOS/FreeRTOS-Cellular-Interface-Reference-Quectel-BG96)
```
Comment (chinglee-iot) [inline README.md:78]:
```suggestion
* [Quectel BG96](https://github.com/FreeRTOS/Lab-FreeRTOS-Cellular-Interface-Reference-Quectel-BG96)
```

File: README.md [hunk 2]
```diff
@@ -82,9 +75,9 @@ It is recommended that you start by cloning the implementation of one of the exi
...
+* [Sierra Wireless HL7802](https://github.com/FreeRTOS/FreeRTOS-Cellular-Interface-Reference-Sierra-Wireless-HL7802)
```
Comment (chinglee-iot) [inline README.md:79]:
```suggestion
* [Sierra Wireless HL7802](https://github.com/FreeRTOS/Lab-FreeRTOS-Cellular-Interface-Reference-Sierra-Wireless-HL7802)
```

File: README.md [hunk 3]
```diff
@@ -82,9 +75,9 @@ It is recommended that you start by cloning the implementation of one of the exi
...
+* [U-Blox Sara-R4](https://github.com/FreeRTOS/FreeRTOS-Cellular-Interface-Reference-ublox-SARA-R4)
```
Comment (chinglee-iot) [inline README.md:80]:
```suggestion
* [U-Blox Sara-R4](https://github.com/FreeRTOS/Lab-FreeRTOS-Cellular-Interface-Reference-ublox-SARA-R4)
```

**Review by chinglee-iot — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ActoryOu) [reply README.md:78]: We'll remove the prefix "Lab-" the repositories when official release.

File: README.md (hunk 2 above)
Comment (ActoryOu) [reply README.md:79]: We'll remove the prefix "Lab-" the repositories when official release.

File: README.md (hunk 3 above)
Comment (ActoryOu) [reply README.md:80]: We'll remove the prefix "Lab-" the repositories when official release.

**Review by chinglee-iot — DISMISSED** [review]

**Review by chinglee-iot — APPROVED** [review]

**Review by chinglee-iot — DISMISSED** [review]

**Review by chinglee-iot — DISMISSED** [review]

**Review by shubnil — APPROVED** [review]

**Comment by paulbartell** [comment]
Can you replace the png with an svg? Large binary objects can over time cause slow clone times with git repositories.

**Review by chinglee-iot — APPROVED** [review]

**Comment by ActoryOu** [comment]
> Can you replace the png with an svg? Large binary objects can over time cause slow clone times with git repositories.

Image resized. Thanks for suggesion.

**Review by paulbartell — APPROVED** [review]

**Pull request merged**